File: Capacitaciones-Front/src/services/entradaService.js
import api from './http'

export const entradaService = {
  buscarPorSeccion: (idSeccion) => api.get(`/entrada/entradaPorSeccion/${idSeccion}`),
  save: (entrada) => api.post('/entrada', entrada),
  eliminar: (id) => api.delete(`/entrada/${id}`),
}

export default entradaService

File: Capacitaciones-Back/capacitaciones/src/main/java/com/example/capacitaciones/service/EntradaService.java
package com.example.capacitaciones.service;

import java.util.List;

import com.example.capacitaciones.DTOs.EntradaDTO;
import com.example.capacitaciones.model.Entrada;

public interface EntradaService {
    Entrada save(EntradaDTO entradaDTO);

    List<EntradaDTO> findAllDTOByIdSeccion(Long idSeccion);

    void delete(Long identrada);

}
